avs-versions

```diff
@@ -17,4 +17,4 @@
 # along with this program. If not, see http://www.gnu.org/licenses/.
 #
 
-export APPSTORE_AVS_VERSION=9.1.13
+export APPSTORE_AVS_VERSION=9.2.2
```

Comment:
is the version public can we update the cartfile to use it too?

Reply:
unfortunately it's not public